src/utils/api.js

```diff
@@ -13,19 +13,26 @@ export default {
   oauthToken: '',
   spaceId: null,
   region: '',
+  client: false,
```

Comment:
Hi @roberto-butti I would not set it to false since the property is meant to be an object. It's better to set it to null initially.

Reply:
Hola @alvarosabu , thank you for your feedback. I adapted the code according to your feedback